### ISSUE TITLE
Anwendung doppelt öffnen?

### ISSUE THREAD

**Issue by Bauer-David** [minova-afis/aero.minova.rcp#1392]
Es wurde irgendwann die Anforderung genannt, dass es die Möglichkeit geben soll, dass der WFC Client doppelt geöffnet wird. Ist diese Anforderung noch aktuell, da dies momentan nicht möglich ist ohne die Kommandozeile.

Falls ja, wäre meine Lösung für das Problem folgendes:

In das Popup Menu im Dock wird ein neuer Menu Punkt "Neue Fenster" wie bei Safari hinzugefügt.

<img width="360" alt="Bildschirmfoto 2022-10-24 um 14 19 43" src="https://user-images.githubusercontent.com/61147549/197524023-cd1bfa41-6da1-480d-8b55-4f573b566e37.png">

Dazu hab ich diesen Stackoverflow Beitrag gefunden:

<img width="681" alt="Bildschirmfoto 2022-10-24 um 14 14 38" src="https://user-images.githubusercontent.com/61147549/197524158-f360650c-d7f8-401c-9c6d-8b81e830588a.png">

**Comment by Bauer-David** [comment]
> Es gibt doch extra die Auswahl des Workspace am Anfang. Wenn man einen anderen Workspace auswählt, müsste es doch funktionieren, wie bei Eclipse auch?

Ja, wenn es möglich wäre die Workspace Auswahl ein zweites Mal zu öffnen, wenn die Anwendung schon läuft

**Comment by wis36** [comment]
Hm, ich hielt das für einen wichtigen Vorteil von Version 12 gegenüber 10/11 🤔
Ich würde mir wünschen dass es noch möglich wird.
Vor allem wenn Kunden vielleicht später ein AFIS 12 + SIS 12 haben, wäre es ungünstig, wenn nur eins gleichzeitig aufgeht.
(Oder wie DISPO + TTA, wie BENOIL)

**Comment by ingraban** [comment]
Lieber David,
unter Windows ist dies ohne Probleme möglich. Dort kann man von jeder Konfiguration eine Instanz gleichzeitig starten.
Unter macOS ist das nicht möglich. Da ist man das auch nicht gewohnt. Aber vielleicht findest Du heraus, wie man es auch dort ermöglichen kann (nur mit Betriebssystembordmitteln).

**Comment by scholz-afis** [comment]
@NEU-Minova hat zu diesem Thema auch schon ein Issue https://github.com/minova-afis/aero.minova.cloud.deployment/issues/80 angelegt.
